src/home/messages.ts

```diff
@@ -0,0 +1,26 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
+  errorMessage: {
+    id: 'category.catalog.error-page-message',
```

Comment:
I see that we start the translation `id` differently. We have two options:
1. `category.catalog...`
2. `catalog...`

Let's choose one option, just `catalog`